src/MessageInput.js

```diff
@@ -0,0 +1,141 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { withStyles } from '@material-ui/core/styles';
+import TextField from '@material-ui/core/TextField';
+import Snackbar from '@material-ui/core/Snackbar';
+import Alert from '@material-ui/lab/Alert';
+
+const api = require('./api.js');
+
+const styles = theme => ({
+  container: {
+    display: 'flex',
+    flexWrap: 'wrap'
+  },
+  textField: {
+    width: '30%',
+    marginRight: 15
+  }
+});
+
+class MessageInput extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      id: `${this.props.id}`,
```

Comment:
Do all of these need to be strings, or can you just set them to `this.props.whatever`?